core/templates/dev/head/pages/exploration_editor/preview_tab/PreviewTab.js

```diff
@@ -112,7 +114,8 @@ oppia.controller('PreviewTab', [
     };
 
     $scope.loadPreviewState = function(stateName, manualParamChanges) {
-      oppiaPlayerService.initSettingsFromEditor(stateName, manualParamChanges);
+      ExplorationPlayerService.initSettingsFromEditor(stateName,
+      manualParamChanges);
```

Comment:
You can move `stateName` also to the next line and indent the line once from `ExplorationPlayerService`, like this
![indent](https://user-images.githubusercontent.com/7807557/31106156-0ab16c44-a807-11e7-9ea2-be926db21d66.jpeg)